viz/pkg/util.go

```diff
@@ -1,6 +1,17 @@
 package pkg
```

Comment:
nit, TIOLI: I see we were already doing this with `util.go`. Maybe move this into a `utils` pkg. `package pkg` feels a bit confusing.